app/controllers/change_requests_controller.rb

```diff
@@ -13,16 +13,28 @@ def create
   end
 
   def create_change_request
+
+    if params[:change_request][:action] == "0"
+      handle_insert
+    elsif params[:change_request][:action] == "1"
+      handle_update 
+    else
+      puts 'unsupported action'
```

Comment:
Maybe this should be a 400 error?

Also, maybe this whole thing could be a `case` statement, and then @azhelle16 could slot her changes in pretty easily by adding a `when` clause for `"2"` (remove).

Reply:
Actually, I take that back; can you make this default to `handle_insert` if the `action` isn't present. This is otherwise going to silently break every single frontend API call to the change requests since they aren't already passing in the `action`.

I'm also open to adding the `action` param to every call, but that's a lot of them to edit on the frontend.

Reply:
yeah I can do that, but if you look at the postman tests a lot of the update client calls will have to be changed anyway